common/src/main/scala/explore/components/Tile.scala

```diff
@@ -117,27 +118,28 @@ object Tile {
               MenuItem(as = <.a)(p.title)
             ),
             p.control.map(b => <.div(ExploreStyles.TileControl, b)),
-            <.span(ExploreStyles.TileTitleStrip,
-                   ExploreStyles.FixedSizeTileTitle.when(!p.canMinimize && !p.canMaximize)
-            ).withRef(infoRef),
+            <.span(^.untypedRef(setInfoRef))(
```

Comment:
I don't recall this on the docs

Reply:
Here we are using a "callback ref", documented here for React: https://reactjs.org/docs/refs-and-the-dom.html#callback-refs

Basically, when a DOM ref is set, a callback is invoked with the value instead of saving it in a mutable var.

The only way I found to have this behavior in scalajs-react is via "untyped ref": https://github.com/japgolly/scalajs-react/blob/master/doc/REFS.md#untyped